### PR TITLE
Add events to Generic Switch XML

### PR DESCRIPTION
**Pull request by jonsmirl**
Generic switch XML missing event definitions, this adds them.

Tested with full regen. No code using these.

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/switch-cluster.xml [hunk 1]
```diff
@@ -27,20 +27,51 @@ limitations under the License.
...
+    </event>
+    <event side="server" code="0x05" priority="info" name="MultiPressOngoing" optional="true">
+      <description>MultiPress Ongoing</description>
+      <field id="0" name="NewPosition" type="INT8U" />
+      <field id="1" name="CurrentNumberOfPressesCounted" type="INT8U" />
+    </event>
+    <event side="server" code="0x06" priority="info" name="MultiPressComplete" optional="true">
+      <description>MultiPress Complete</description>
+      <field id="0" name="NewPosition" type="INT8U" />
+      <field id="1" name="CurrentNumberOfPressesCounted" type="INT8U" />
```
Comment (Damian-Nordic) [inline src/app/zap-templates/zcl/data-model/chip/switch-cluster.xml:64]:
```suggestion
      <field id="1" name="TotalNumberOfPressesCounted" type="INT8U" />
```

**Review by Damian-Nordic — APPROVED** [review]
Approving provided that MultiPress Complete will be fixed.

**Comment by andy31415** [comment]
@jonsmirl it looks like zap updates are needed (or restyle diff)

**Review by andy31415 — APPROVED** [review]

**Comment by andy31415** [comment]
fast track: xml  changes only

**Comment by jonsmirl** [comment]
> @jonsmirl it looks like zap updates are needed (or restyle diff)

Yes, it need regen before committing. But if I keep regening as top of tree changes I trigger a large amount of pointless CI jobs.

**Pull request merged**